packages/paste-core/core-bundle/rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import typescript from 'rollup-plugin-typescript2';
+import babel from 'rollup-plugin-babel';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+// import {terser} from 'rollup-plugin-terser';
```

Comment:
Do we need this commented line for something in the future?

Reply:
Oops, I forgot to un-comment it after testing my dist output. Thanks!